src/jdk.jfr/share/classes/jdk/jfr/internal/PlatformEventType.java

```diff
@@ -180,8 +192,8 @@ public boolean hasStackTrace() {
     }
 
     public boolean hasThreshold() {
-        if (hasCutoff) {
-            // Event has a duration, but not a threshold. Used by OldObjectSample
+        if (hasCutoff || isCPUTimeMethodSampling) {
```

Comment:
This is not a duration event anymore, right?

Reply:
Yes